package.json

```diff
@@ -19,12 +19,19 @@
   "repository": "github:sandstreamdev/eoc",
   "dependencies": {
     "babel-polyfill": "6.26.0",
+    "body-parser": "1.18.3",
     "classnames": "2.2.6",
+    "cors": "2.8.5",
     "express": "4.16.3",
     "mongodb": "3.1.10",
+    "mongoose": "5.3.13",
     "prop-types": "15.6.2",
     "react": "16.5.2",
-    "react-dom": "16.5.2"
+    "react-dom": "16.5.2",
+    "react-redux": "5.1.1",
+    "redux": "4.0.1",
+    "redux-devtools-extension": "2.13.7",
```

Comment:
this should be in dev dependencies

Reply:
I agree but EsLint not :-1:  
![screenshot from 2018-12-06 15-48-58](https://user-images.githubusercontent.com/27632432/49591359-89e4eb80-f96e-11e8-8385-acef881cb3ff.png)

Reply:
Please check:
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md